roles/check_config_dynamic/tasks/main.yml

```diff
@@ -1,36 +1,26 @@
 ---
 
-- name: disk space check - fail when disk is full
+- name: Disk space check - Fail task when disk is full
```

Comment:
This task is really needed in both check_config_dynamic and check_system_necessary roles?

Reply:
start.yml/stop.yml and other playbooks will always include check_config_dynamic role.
Only bootstrap.yml include check_system_necessary role. it is said the task is a corner case.